plugins/system-update/click/apiclient_impl.cpp

```diff
@@ -78,33 +83,26 @@ void ApiClientImpl::requestMetadata(const QUrl &url,
 
 void ApiClientImpl::requestUpdatesMetadata(const QStringList &packages)
 {
-    QUrl url(Helpers::clickMetadataUrl());
-    QUrlQuery query(url);
-
-    QJsonObject body;
-
-    QStringList frameworks = Helpers::getAvailableFrameworks();
-    body.insert("frameworks", QJsonArray::fromStringList(frameworks));
-
-    body.insert("architecture",
-                QString::fromStdString(Helpers::getArchitecture()));
-
-    body.insert("apps", QJsonArray::fromStringList(packages));
-
-    body.insert("channel", Helpers::getSystemCodename());
-
-    QJsonDocument doc(body);
-    QByteArray content = doc.toJson();
-
-    QUrl u(url);
-    u.setQuery(query);
-    QNetworkRequest request;
-    request.setHeader(QNetworkRequest::ContentTypeHeader, "application/json");
-    request.setUrl(u);
-    request.setOriginatingObject(this);
-    request.setAttribute(QNetworkRequest::User, "metadata-request");
-
-    initializeReply(m_nam->post(request, content));
+    m_hasErrors = false;
```

Comment:
please also set `m_requests` to 0 here, it's safer. Maybe something like this:

    if (m_requests != 0) {
        qCritical() << Q_FUNC_INFO << "Has still some requests active:" << m_requests;
        m_requests = 0;
    }